.github/workflows/vulnerability.yml

```diff
@@ -8,15 +8,6 @@ name: Vulnerability Scope Analyzer
 # or API.
```

Comment:
Workflow builds are public, we shouldn't have this file at all, as it displays all the environment variables, posing a security vulnerability already.

Reply:
Done in `testing-workflows`